Sources/GenericModule/View/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright © 2021 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+open class ViewController<ViewModel: GenericModule.ViewModel, ViewInput, Output>: UIViewController, View {
+    public var output: Output
+    public var viewModel: ViewModel
+
+    private var genericModuleViewOutput: ViewOutput? {
+        output as? ViewOutput
+    }
```

Comment:
```suggestion
open class ViewController<ViewModel: GenericModule.ViewModel, ViewInput, Output: ViewOutput>: UIViewController, View {
    public var output: Output
    public var viewModel: ViewModel
```

Reply:
Unfortunately, it makes impossible to use protocols as Output on user side code